### ISSUE TITLE
Accessibility - iOS - English - Search Button Does Not Announce Function

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#118]
When using VoiceOver, it doesn't announce that the "Search" button is for searching. The application used to tell the user that it was for search and you had to double tap to open.

Currently, VoiceOver announces the following:

"Toggle button not pressed double tap to toggle settings."

The text should probably be "Search toggle button not pressed double tap to toggle settings."

I assume the same thing is happening for the Spanish and French versions too so we will need to have that translated. If so I will open an issue for each language.
